extensions/sql/contract-definition/store/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2022 Daimler TSS GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Daimler TSS GmbH - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+    `java-test-fixtures`
+    `maven-publish`
+}
+
+dependencies {
+    api(project(":spi"))
+    api(project(":extensions:dataloading"))
+    api(project(":extensions:transaction:transaction-spi"))
+    api(project(":extensions:transaction:transaction-datasource-spi"))
+
```

Comment:
Do all the above need to be API dependencies? Why not impl?

Reply:
I think API scope is appropriate when the current module exposes contained SPI classes with its public signature, otherwise it is an implementation detail.